### PR TITLE
Added min and max value functions to extract the maximum or minimum v…

### PR DESCRIPTION
**Pull request by santiagoeid93**
Created min and max functions to extract the minimum/maximum value within arrays. Functions filter out anything that isn't a number from arrays for improved functionality.

### CODE REVIEW

File: src/Helpers.js [hunk 1]
```diff
@@ -205,7 +205,22 @@ const evalWithSafeEnvironment = (function () {
 		var elements = Array.prototype.slice.call(arguments);
 		return elements.join('');
 	}
-
+	function max(array) {
+		if (array != 0) {
```
Comment (maurotrigo) [inline src/Helpers.js]: To check if a variable is an array, please use ```Array.isArray() ```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

So, the condition should be:

```if (Array.isArray(array) && array.length > 0)```

File: src/Helpers.js [hunk 2]
```diff
@@ -205,7 +205,22 @@ const evalWithSafeEnvironment = (function () {
 		var elements = Array.prototype.slice.call(arguments);
 		return elements.join('');
 	}
-
+	function max(array) {
```
Comment (maurotrigo) [inline src/Helpers.js]: The argument name ```array``` could lead to confusion and errors, since it's a generic term in most programming languages (and a reserved word in some of them). Also, at the beginning of the function, it's not clear that the variable is in fact an array. Please use another variable name, like *value*.

File: src/Helpers.js [hunk 3]
```diff
@@ -205,7 +205,22 @@ const evalWithSafeEnvironment = (function () {
...
+		if (array != 0) {
+			var maxFilteredArray = _.max(array);
+			return maxFilteredArray;
```
Comment (maurotrigo) [inline src/Helpers.js]: In this case, it's not necessary to assign the result to a variable, since it's not being evaluated and the impact on legibility is small. It's better to return the result of _.max directly.

File: src/Helpers.js (hunk 2 above)
Comment (santiagoeid93) [reply src/Helpers.js]: array has been changed to value as specified.

File: src/Helpers.js (hunk 1 above)
Comment (santiagoeid93) [reply src/Helpers.js]: Conditional statement modified. Thanks!

File: src/Helpers.js (hunk 3 above)
Comment (santiagoeid93) [reply src/Helpers.js]: Proposed change done!

**Review by maurotrigo — APPROVED** [review]

**Review by maurotrigo — APPROVED** [review]

**Review by jlarteaga — CHANGES_REQUESTED** [review]
I suggest to include tests for numbers, null and undefined values, and strings, so the behaviour of both functions is completely detailed.

**Comment by santiagoeid93** [comment]
@jlarteaga I added the specified tests and changed the return values from boolean to null as adviced. The changes are included in the third commit.

**Review by jlarteaga — APPROVED** [review]
